omnibus/package-scripts/updater-rpm/posttrans

```diff
@@ -66,6 +68,7 @@ chmod 750 ${HELPER}
 setcap cap_setuid+ep ${HELPER}
 
 $BOOTSTRAP_INSTALLER bootstrap --url "oci://docker.io/datadog/installer-package-dev:latest"
+$STABLE_INSTALLER bootstrap
```

Comment:
this will break until release of a new OCI given that the bootstrap command.
Adding a || true and skipping the test checking systemd installer unit could be a temp solution, releasing it in 2 steps

Reply:
It's ok as the e2e tests are [targeting the boot installer for now](https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/tests/updater/linux_test.go#L120)

Baptiste is working on building a temp CR for tests, and we can have the apt/rpm script next check the config to see from which CR to install from